corehq/apps/app_execution/static/app_execution/js/workflow_charts.js

```diff
@@ -1,102 +1,97 @@
+import "commcarehq";
+import nv from "jquery";
+import "moment/moment";
```

Comment:
Would this address the `'moment' is not defined` lint errors?
```suggestion
import moment from "moment/moment";
```

Reply:
yes, thank you